.github/workflows/assign-for-verification.yml

```diff
@@ -0,0 +1,26 @@
+name: Verify Install
+on:
+  issues:
+    types: [labeled]
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v4
+        with:
+          script: |
+            const label = context.payload.label
+            if (label && label.name === "3 - installed") {
+              github.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: "This issue is installed, assigning to @benelan for verification",
+              });
+              await github.issues.update({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                assignees: ['benelan']

```

Comment:
Nitpick: can you use double quotes for consistency? Sidebar, looks like we also need to include `yml` files in our linting.

Reply:
I did a find/replace for all of the yml workflow files, I'm a nitpicker too. I'll look into the linting.